packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -142,5 +142,6 @@ export interface FeatureToggles {
   kubernetesPlaylists?: boolean;
   navAdminSubsections?: boolean;
   recoveryThreshold?: boolean;
+  teamHeaders?: boolean;
```

Comment:
Let's maybe do it more specific
```suggestion
  teamHttpHeaders?: boolean;
```

Reply:
agree to be more specific on the feature toggle naming here